chapters/planning-data-work.tex

```diff
@@ -1,147 +1,457 @@
+some kind of \textbf{file sharing} software.
+  \index{file sharing}
+The exact services you use will depend on your tasks,
+but in general, there are different approaches to file sharing, and the three discussed here are the most common.
+\textbf{File syncing} is the most familiar method,
+and is implemented by software like Dropbox and OneDrive.
+  \index{file syncing}
+Sync forces everyone to have the same version of every file at the same time,
+which makes simultaneous editing difficult but other tasks easier.
+They also have some security concerns which we will address later.
+\textbf{Version control} is another method,
+commonly implemented by tools like Git and GitHub.
+  \index{version control}
+Version control allows everyone to access different versions of files at the same time,
+making simultaneous editing easier but some other tasks harder.
+It is also only optimized for specific types of files.
+Finally, \textbf{server storage} is the least-common method,
+because there is only one version of the materials,
+and simultaneous access must be carefully regulated.
+  \index{server storage}
+Server storage ensures that everyone has access
+to exactly the same files and environment, and it also enables
+high-powered computing processes for large and complex data.
+All three file sharing methods are used for collaborative workflows,
+and you should review the types of data work
+that you will be doing, and plan which types of files
+will live in which types of sharing services.
+It is important to note that they are, in general, not interoperable:
+you cannot have version-controlled files inside a syncing service,
+or vice versa, without setting up complex workarounds,
+and you cannot shift files between them without losing historical information.
+Therefore, choosing the correct sharing service at the outset is essential.
+
+% ----------------------------------------------------------------------------------------------
+\subsection{Documenting decisions and tasks}
+
+Once your technical and sharing workspace is set up,
+you need to decide how you are going to communicate with your team.
+The first habit that many teams need to break
+is using instant communication for management and documentation.
+Email is, simply put, not a system. It is not a system for anything. Neither is WhatsApp.\index{email}\index{WhatsApp}
+These tools are developed for communicating ``now'' and this is what they do well.
+They are not structured to manage group membership or to present the same information
+across a group of people, or to remind you when old information becomes relevant.
+They are not structured to allow people to collaborate over a long time or to review old discussions.
+It is therefore easy to miss or lose communications from the past when they have relevance in the present.
+Everything that is communicated over e-mail or any other instant medium should
+immediately be transferred into a system that is designed to keep records.
```

Comment:
I think we are more convincing in the point we are trying to make if we were more specific here. I understand that what we mean here is not that every single email - no matter how irrelevant - should be transferred to another system, but if one read literary what we currently say, then that is what we are saying.

I guess the point is that once one get an email one should make a decision if that information will ever be relevant for anyone in the future. If not (like an email about when to meet next week), then it is fine to keep it only as an email. But if it is (like an email about dropping a part of the sample) then that information should be moved to a less instant and individual based system. That is not the best way to say it, but do you guys agree that that is the point we are making?

Reply:
5719c82dbeb25511551a1a2d465f32055a095d11